superset/db_engines/hive.py

```diff
@@ -30,8 +30,9 @@ def fetch_logs(self, max_rows=1024,
             )
             response = self._connection.client.FetchResults(req)
             hive._check_status(response)
-            assert not (
-                response.results.rows, 'expected data in columnar format'
+            assert (
+                (not response.results.rows or len(response.results.rows) == 0),
```

Comment:
Just a "nulls" vs "things that define __len__" not begin the same .. so should `resp` return an object (that's not null, but empty) .. lots of folks forget the `__nonzero__` method is a thing.